types/tslint.json

```diff
@@ -0,0 +1,14 @@
+{
+  "extends": "dtslint/dtslint.json",
+  "rules": {
+    "callable-types": false,
+    "max-line-length": false,
+    "no-redundant-jsdoc": false,
+    "no-void-expression": false,
+    "only-arrow-functions": false,
```

Comment:
when enabled, these rules aren't flagging any issues, we can safely re-enable them.